exist-core/src/main/java/org/exist/util/Configuration.java

```diff
@@ -260,6 +260,12 @@ public Configuration(String configFilename, Optional<Path> existHomeDirname) thr
                 configureTransformer((Element)transformers.item(0));
             }
 
+            //saxon settings (most importantly license file for PE or EE features)
+            final NodeList saxon = doc.getElementsByTagName(SaxonConfigurationHolder.CONFIGURATION_ELEMENT_NAME);
```

Comment:
Here you can see clearly that the name is too generic and clashes with other `CONFIGURATION_ELEMENT_NAME`s